action.yml

```diff
@@ -1,76 +1,40 @@
 # action.yml
-name: 'lazy-action-dotnet'
-description: 'Github action to perform build, test , scan and generate image.'
+name: "lazy-action-dotnet"
+description: "Github action to perform build, test , scan and generate image."
 inputs:
-  src_file_dir_path:  # source file directory path
-    description: 'Source file directory'
-    required: true
-    default: '.'
-  dockerfile_dir_path:  # docker file directory path
-    description: 'Dockerfile  directory'
-    required: true
-    default: '.'
-  github_token:  # github owner token
-    description: 'Github token'
-    required: true
-  github_owner_token:  # github owner token
-    description: 'Github owner token'
-    required: true
-  sonar_token:  # sonar token
-    description: 'Github token'
-    required: true
-  aws_access_key_id:  # aws access key id
-    description: 'AWS access key'
-    required: true
-  aws_secret_access_key:  # aws secret access key
-    description: 'AWS secret access key'
-    required: true
-  aws_region:  # aws region
-    description: 'AWS Region'
+  src_file_dir_path: # source file directory path
```

Comment:
Remove comments from this file since there is a description. Make sure that the readme and the description have the same text

Reply:
Removed and updated with same description.